README.md

```diff
@@ -8,40 +8,57 @@
 `npm install lazerpay-node-sdk`
 
 ```javascript
-const LazerPay = require('lazerpay-node-sdk');
+const Lazerpay = require('lazerpay-node-sdk');
 
-const lazerpay = new LazerPay(LAZER_PUBLIC_KEY);
+const lazerpay = new Lazerpay(LAZER_PUBLIC_KEY);
 ```
 
 For staging, Use TEST API Keys and for production, use LIVE API KEYS.
 You can get your LAZER_PUBLIC_KEYS from the Lazerpay dashboard.
```

Comment:
Change to `Use TEST API keys for testing, and LIVE API keys for production`. 

Reason: Better English, and there's no guarantee that the user has "staging" as environment

Reply:
cool